client/src/main/webjar/organization-list/organization-list.component.js

```diff
@@ -0,0 +1,64 @@
+(function (angular) {
+    "use strict";
+
+    var module = angular.module("healthBam.organizationList");
```

Comment:
Missing a `client/src/test/webjar/organization-list/organization-list.component.spec.js` from this pull-request.